lib/styles/components/profile-settings.less

```diff
@@ -26,7 +26,7 @@
                 }
             }
             &::before {
-                content: "Change\00000aAvatar";
+                content: attr(data-changeavatartextlineone) " \00000a " attr(data-changeavatartextlinetwo);
```

Comment:
I am not sure that this will work. Quote characters look somehow wrong.

Reply:
Please test this.

Reply:
Tested Working Fine